src/geo/pelias.js

```diff
@@ -0,0 +1,43 @@
+import request from 'superagent'
```

Comment:
Can you put this in a top-level folder (maybe `lib` or `internal`?) since the `src` folder is directly scaffolded into the user-facing API? This should be kept internal so we don't have a new API.

Reply:
Yes, this seems like a much cleaner solution.